contrib/utilities/run_clang_tidy.sh

```diff
@@ -40,28 +40,12 @@ if test ! -d "$SRC/source" -o ! -d "$SRC/include" -o ! -d "$SRC/examples" -o ! -
 fi
 echo "SRC-DIR=$SRC"
 
-# do not allow bundled packages, otherwise we get too many warnings from TBB/UMFPACK/etc.
 # enable MPI (to get MPI warnings)
 # export compile commands (so that run-clang-tidy.py works)
-ARGS="-D DEAL_II_ALLOW_BUNDLED=OFF -D DEAL_II_WITH_MPI=ON -D CMAKE_EXPORT_COMPILE_COMMANDS=ON $@"
+ARGS="-D DEAL_II_WITH_MPI=ON -D CMAKE_EXPORT_COMPILE_COMMANDS=ON $@"
```

Comment:
ShellCheck says:
```
^-- SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```
I don't know how to improve this though. I am fine with ignoring it.